CMakeLists.txt

```diff
@@ -6,33 +6,16 @@
 # SPDX-License-Identifier: MIT
 #
 
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 3.0)
+cmake_policy(VERSION 3.0)
 
-cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
+project(rgbds
+        LANGUAGES C)
 
-set(RGBDS_VER 0.4.1)
-set(RGBDS_DESC "Rednex Game Boy Development System")
-
-if(CMAKE_VERSION VERSION_LESS 3.0)
-  project(rgbds C)
-  set(PROJECT_VERSION "${RGBDS_VER}")
-else()
-  if(CMAKE_VERSION VERSION_LESS 3.9)
-    project(rgbds VERSION "${RGBDS_VER}"
-                  LANGUAGES C)
-  else()
-    project(rgbds VERSION "${RGBDS_VER}"
-                  DESCRIPTION "${RGBDS_DESC}"
-                  LANGUAGES C)
-  endif()
-endif()
-
-if(CMAKE_VERSION VERSION_LESS 3.9)
-  set(PROJECT_DESCRIPTION "${RGBDS_DESC}")
-endif()
+option(DEVELOP "build in development mode" OFF)
 
 set(DEFAULT_BUILD_TYPE "Release")
-if(NOT CMAKE_BUILD_TYPE)
+if(NOT CMAKE_BUILD_TYPE AND NOT DEVELOP)
```

Comment:
I would set the DEFAULT_BUILD_TYPE to "Debug" when the DEVELOP flag is active, instead of defaulting to "empty" (which is what CMake does by default)

Reply:
Good idea. Done.